test/gate/test_coercer.rb

```diff
@@ -6,6 +6,13 @@ def test_coerce_string
       assert_equal "o Hai", coercer_for(:String).coerce("o Hai")
     end
 
+    def test_coerce_any
+      assert_equal "o Hai", coercer_for(:Any).coerce("o Hai")
+      assert_equal [1, 2, 3] ,coercer_for(:Any).coerce([1, 2, 3])
+      assert_equal false ,coercer_for(:Any).coerce(false)
+      assert_equal true ,coercer_for(:Any).coerce(true)
```

Comment:
Please reformat into `something, coercer_for(...)` (space after coma, not before ;))